config/flags/flags.go

```diff
@@ -307,6 +307,11 @@ var (
 		Usage: "Limit maximum size in some RPC calls execution",
 		Value: gossip.DefaultConfig(cachescale.Identity).MaxResponseSize,
 	}
+	StructLogLimitFlag = cli.IntFlag{
```

Comment:
Looks like this flag is not used anywhere ... I guess it should be integrated somewhere in a list of flags and read?

Reply:
You are right, thanks for noticing